src/frontend/src/icp-eth/components/tokens/AddTokenByNetwork.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import { i18n } from '$lib/stores/i18n.store';
+	import { Dropdown, DropdownItem } from '@dfinity/gix-components';
+	import { networksMainnets } from '$lib/derived/networks.derived';
+	import { ETHEREUM_NETWORK_ID, ICP_NETWORK_ID } from '$env/networks.env';
+	import IcAddTokenForm from '$icp/components/tokens/IcAddTokenForm.svelte';
+	import AddTokenForm from '$eth/components/tokens/AddTokenForm.svelte';
+	import { fade } from 'svelte/transition';
+	import type { Network } from '$lib/types/network';
+	import { nonNullish } from '@dfinity/utils';
+
+	export let network: Network | undefined;
+	export let tokenData: Record<string, string>;
+
+	let networkName: string | undefined = network?.name;
+	$: networkName,
+		(network = nonNullish(networkName)
+			? $networksMainnets.find(({ name }) => name === networkName)
+			: undefined);
+
+	let ledgerCanisterId: string;
+	let indexCanisterId: string;
+	let erc20ContractAddress: string;
+
+	$: tokenData = {
+		ledgerCanisterId,
+		indexCanisterId,
+		erc20ContractAddress
+	};
+</script>
+
+<div class="stretch pt-8">
+	<label for="network" class="font-bold px-4.5">{$i18n.tokens.manage.text.network}:</label>
+
+	<div id="network" class="mb-4 mt-1 pt-0.5">
+		<Dropdown name="network" bind:selectedValue={networkName}>
+			<option disabled selected value={undefined} class="hidden"
+				><span class="description">{$i18n.tokens.manage.placeholder.select_network}</span></option
+			>
+			{#each $networksMainnets as network}
```

Comment:
Why not Sepolia as well? User can set up custom tokens on this particular chain.

Reply:
no particular reason to be honest, ii think it was an overview